doc/api/net.md

```diff
@@ -1729,7 +1729,7 @@ net.isIPv6('fhqwhgads'); // returns false
 [`'data'`]: #event-data
 [`'drain'`]: #event-drain
 [`'end'`]: #event-end
-[`'error'`]: #event-error_1
+[`'error'`]: #event-error-1
```

Comment:
The current anchor exists in our HTML docs. The new one proposed here does not. What is the motivation for this change? Is it to fix something in the GitHub markdown rendering of this doc?